src/polyfill.native.ts

```diff
@@ -9,14 +9,14 @@ class MediaQuery {
   private listeners: Listener[] = [];
 
   private orientation: ScreenOrientation.Orientation =
-    ScreenOrientation.Orientation.PORTRAIT;
+    ScreenOrientation.Orientation.PORTRAIT_UP;
```

Comment:
Shouldn't here be assigned just a string "portrait", as later we are asigning also a string? (L55-59)

Reply:
Ah yes definitely, it should be just `"portrait"`. It was working anyway when I tested it though, were you getting errors?

Reply:
Actually I just checked again and I think it's fine the way it is. We aren't assigning a string later in L55-59, we're passing the string to `mediaQuery.match` after comparing `this.orientation` to `ScreenOrientation.Orientation`.